CommunityGuidelines.md

```diff
@@ -0,0 +1,61 @@
+# Community Guidelines
+
+## Community rules
+
+- Please be respectful to other members. Avoid any racist, sexist, homophobic, excessively political, or otherwise distasteful discussions.
+- While some cursing is appropriate in context, refrain from using excessive profanity.
+- Keep conversations in the appropriate channel. Some off-topic discussions are to be expected but try not to blatantly post non-subject-related content.
+- No NSFW material is allowed. Breaking this rule will be met with an immediate ban.
```

Comment:
Does it make sense to not use the NSFW acronym here?

Reply:
NSFW would intend anything not safe for work, so anything lewd, gore or other things deemed inappropriate. I do think that makes sense to use it here. If you rather have it listed as `No lewd, gore or other things deemed inappropriate allowed. Breaking this rule will be met with an immidate ban.` That could work too, ofcourse. Would clarify it better perhaps.

Reply:
I'm pretty sure that Tuluo knows that, but his question is about whether to use the acronym or the full meaning. In papers and official documents, acronyms are only used after being explicitly states once.

```suggestion
- No not safe for work (NSFW) material is allowed. Breaking this rule will be met with an immediate ban.
```